indico/web/client/js/react/components/Datepicker.jsx

```diff
@@ -0,0 +1,80 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {Param, Translate} from 'indico/react/i18n';
+
+import 'indico/custom_elements/ind_datepicker';
+
+export default function Datepicker({format, onChange, value, ...inputProps}) {
+  function handleDateChange(ev) {
+    const {date} = ev.target.closest('ind-date-picker');
+    onChange(date?.toLocaleDateString('en-CA'));
```

Comment:
This may deserve a comment since I guess you use en-CA because they us `YYYY-MM-DD` for dates, but someone not knowing this will probably wonder why we use en-CA here